cubical/solver.py

```diff
@@ -8,16 +8,22 @@
 import numpy as np
 import traceback
 from cubical.tools import logger, ModColor
-from cubical.data_handler import FL, Tile
+from cubical.data_handler import Tile
+from cubical.flagging import FL
 from cubical.machines import jones_chain_machine
 from cubical.statistics import SolverStats
+from pdb import set_trace as BREAK  # useful: can set static breakpoints by putting BREAK() in the code
```

Comment:
This should probably be removed once debugging is finished.

Reply:
> ***once*** debugging is finished

Ahh, so young, so optimistic. This is exactly why this cynical old man is leaving it in. **chortle**

(Did you ever notice the Dutch have difficulty in using *if* versus *when* properly?)

Reply:
Hahaha. :+1: